src/lab/Graphs/RadialGraphs/RadialProgressChart/testData.ts

```diff
@@ -3,7 +3,7 @@ import { RadialChartMetric } from "../base";
 const testRadialProgressChartData: RadialChartMetric = {
   value: 60,
   label: "pass",
-  baseColor: "#5B44BA",
+  baseColor: "orange",
```

Comment:
I don't understand this change but it's test data so I think it's fine

Reply:
This hex code was the same as the purple theme colour of litmus. So the change in icon background of the RadialProgressChart was not distinguishable. Now by testing out a completely different colour which is not part of the litmus theme colour we are able to know that the prop `baseColor` is having its effect on both the colour of the Arc and the icon